system/java_certs.py

```diff
@@ -0,0 +1,218 @@
+    required: false
+    default: 443
+  cert_path:
+    description:
+      - Local path to load certificate from
+    required: false
+    default: null
+  cert_alias:
+    description:
+      - Imported certificate alias
+    required: false
+    default: null
+  keystore_path:
+    description:
+      - Path to keystore.
+    required: false
+    default: null
+  keystore_pass:
+    description:
+      - Keystore password
+    required: false
+    default: null
+  state:
+    description:
+      - Defines action which can be either certificate import or removal.
+    choices: [ 'present', 'absent' ]
+    default: 'present'
+    required: false
+
+author: Adam Hamsik
+'''
+
+EXAMPLES = '''
+# Import SSL certificate from google.com to a given cacerts keystore
+java_certs: cert_url=google.com cert_port=443 keystore_path=/usr/lib/jvm/jre7/lib/security/cacerts keystore_pass=changeit state=present
+
+# Remove certificate with given alias from a keystore
+java_certs: cert_url=google.com keystore_path=/usr/lib/jvm/jre7/lib/security/cacerts keystore_pass=changeit state=absent
+'''
+
+RETURN = '''
+msg:
+  description: Output from stdou of keytool command after execution of given command.
+  returned: success
+  type: string
+  sample: "Module require existing keystore at keystore_path '/tmp/test/cacerts'"
+
+rc:
+  description: Keytool command execution return value
+  returned: success
+  type: int
+  sample: "0"
+
+cmd:
+  description: Executed command to get action done
+  returned: success
+  type: string
+  sample: "keytool -importcert -noprompt -keystore"
+'''
+
+try:
+    import json
+except ImportError:
+    import simplejson as json
+
+import os
+import subprocess
+import sys
+import datetime
+import syslog
+
+def check_cert_present(module, keystore_path, keystore_pass, alias):
+    test_cmd = "keytool -noprompt -list -keystore '%s' -storepass '%s' -alias '%s'" % (keystore_path, keystore_pass, alias)
+
+    (rc, out, err) = module.run_command(test_cmd)
+    if rc == 0:
+        return True
+    return False
+
+def import_cert_url(module, url, port, keystore_path, keystore_pass, alias):
+    fetch_cmd = "keytool -printcert -rfc -sslserver %s:%s" % (url, port)
+    import_cmd = "keytool -importcert -noprompt -keystore '%s' -storepass '%s' -alias '%s'" % (keystore_path, keystore_pass, alias)
+
+    # Fetch SSL certificate from remote host.
+    (rc, fetch_out, fetch_err) = module.run_command(fetch_cmd, check_rc=True)
+
+    # Use remote certificate from remote host and import it to a java keystore
+    (rc, import_out, import_err) = module.run_command(import_cmd, data=fetch_out, check_rc=False)
+    if rc == 0:
+        return module.exit_json(changed=True, msg=import_out,
+            rc=rc, cmd=import_cmd, stdout_lines=import_out)
+    else:
+        return module.fail_json(msg=import_out, rc=rc, cmd=import_cmd)
+
+
+def import_cert_path(module, path, keystore_path, keystore_pass):
```

Comment:
I don't follow here, you would like to have alias argument in import_cert_path because alias is not used there. I can add it there if it would help you.